scripts/machine-launch-script.sh

```diff
@@ -76,8 +175,11 @@ sudo python3 -m pip install pyyaml==5.4.1
 # setup argcomplete
 activate-global-python-argcomplete
 
-} 2>&1 | tee /home/centos/machine-launchstatus.log
+} 2>&1 | tee $LAUNCH_STATUS_PATH/machine-launchstatus.log
+
+if [ $FLAVOR == "ubuntu" ]; then
+    # default /bin/sh to bash (not dash)
+    sudo dpkg-reconfigure -p critical dash
+fi
```

Comment:
As a matter of personal preference, I like this and always do it on Ubuntu machines that I admin.  However, I don't think you guys should _require_ this.  Anywhere that you are using `/bin/sh` and really want it to be BASH, you should really use `/bin/bash`.